lib/runcible/models/yum_importer.rb

```diff
@@ -3,6 +3,13 @@ module Models
     class YumImporter < Importer
       ID = 'yum_importer'
       REPO_TYPE = 'rpm-repo'
+      DOWNLOAD_POLICIES = [
+        DOWNLOAD_IMMEDIATE = 'immediate',
+        DOWNLOAD_ON_DEMAND = 'on_demand',
+        DOWNLOAD_BACKGROUND = 'background'
```

Comment:
I think that would help me (so i assume others?) not get stuck on trying to interpret this chunk of code to change it :)